pulp_container/app/migrations/0010_rbac_for_repositories.py

```diff
@@ -0,0 +1,85 @@
+# Generated by Django 2.2.17 on 2020-11-17 13:17
+
+from django.db import migrations
+
+from pulpcore.plugin.models import AccessPolicy
+
+
+REPOSITORY_STATEMENTS = [
+    {
+        "action": ["list"],
+        "principal": "authenticated",
+        "effect": "allow",
+    },
+    {
+        "action": ["create"],
+        "principal": "authenticated",
+        "effect": "allow",
+        "condition": "has_model_perms:container.add_containerrepository",
+    },
+    {
+        "action": ["retrieve"],
+        "principal": "authenticated",
+        "effect": "allow",
+        "condition": "has_model_or_obj_perms:container.view_containerrepository",
+    },
+    {
+        "action": ["destroy"],
+        "principal": "authenticated",
+        "effect": "allow",
+        "condition": "has_model_or_obj_perms:container.delete_containerrepository",
+    },
+    {
+        "action": ["update", "partial_update"],
+        "principal": "authenticated",
+        "effect": "allow",
+        "condition": "has_model_or_obj_perms:container.change_containerrepository",
+    },
+    {
+        "action": ["sync"],
+        "principal": "authenticated",
+        "effect": "allow",
+        "condition": [
+            "has_model_or_obj_perms:container.modify_repo_content",
+            "has_remote_param_model_or_obj_perms:container.view_containerremote",
```

Comment:
This probably means that we depend on implementing the policy for Remotes first.